templates/zerver/help/custom-certificates.md

```diff
@@ -35,6 +41,62 @@ administrator.
 2. Select the **Organizations** tab.
 
 3. Under **Add Custom Certificates**, enter your organization URL and add
-   the custom certificate file (should end in `.crt` or `.pem`).
+   the custom certificate file (it should end in `.crt` or `.pem`).
 
 {end_tabs}
+
+
+# Adding a certificate
+
+A file containing the certificate will need to be provided by your
+administrator; it should be a file ending in `.pem` or `.crt`.
+
+
+{start_tabs}
+{tab|mac}
+1. Hit `Cmd+Space` to bring up Spotlight Search, type `Keychain
+   Access`, and press `Enter`.
+
+2. From the `File` menu, choose `Import Items...`
+
+3. Navigate to the certificate file, then click `Open`.
+
+4. Right-click the newly-added certificate, and click `Get Info` from
+   the context menu.
+
+5. Expand the `Trust` section.
+
+6. Select `Always Trust` for the `Secure Sockets Layer (SSL)` option.
+
+7. Close the window.  You will be prompted for your password to verify
+   the change.
+
+{tab|windows}
+1. Open Google Chrome, and open the URL `chrome://settings/`.
```

Comment:
I expect typical users would find it simpler to go to the Chrome menu (⋮) → Settings?